bin/setup

```diff
@@ -36,6 +36,10 @@ Dir.chdir APP_ROOT do
   system "rm -f log/*"
   system "rm -rf tmp/cache"
 
+  puts "\n== Creating styles/_variables.scss =="
```

Comment:
Let's go ahead and convert all of these system calls to FileUtils, Dir, or File to make them more platform agnostic.  Also, let's do a check to see if the file is there and not create or copy if it is.